### ISSUE TITLE
Multiline string with `<<` not caught by `Style/LineEndConcatenation`

### ISSUE THREAD

**Issue by mockdeep** [rubocop/rubocop#1474]
The following string passes the `LineEndConcatenation` cop:

``` ruby
flash[:error] = 'Here is a string ' \
                'That spans' <<
  'multiple lines'
```
